lib/tic_tac_toe.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+# ./lib/TicTacToe.rb
+
+class TicTacToe
+  attr_reader :board
+  attr_reader :players
+  attr_reader :turn
+  attr_reader :status
+  attr_reader :enum_status
```

Comment:
[OPTIONAL]
This can be condensed into one line like so:
`attr_reader :board, :players etc`